packages/flutter/lib/src/widgets/autocomplete.dart

```diff
@@ -269,6 +271,14 @@ class RawAutocomplete<T extends Object> extends StatefulWidget {
   /// This parameter is ignored if [textEditingController] is defined.
   final TextEditingValue? initialValue;
 
+  /// If the options view overlay should be shown when no options were returned
+  /// from [optionsBuilder].
```

Comment:
Should the docs for these parameters be the same as in Autocomplete? Maybe use a macro, or just copy the text.

Reply:
Changed to use a macro